2020-InfiniteRecharge/src/main/java/frc/robot/subsystems/Chassis.java

```diff
@@ -1,64 +1,188 @@
 package frc.robot.subsystems;
 
+import javax.rmi.ssl.SslRMIClientSocketFactory;
+
 import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.revrobotics.CANPIDController;
 import com.revrobotics.CANSparkMax;
+import com.revrobotics.ControlType;
 import com.revrobotics.CANSparkMaxLowLevel.MotorType;
 
 import edu.wpi.first.wpilibj.Joystick;
 import edu.wpi.first.wpilibj.SpeedControllerGroup;
 import edu.wpi.first.wpilibj.drive.DifferentialDrive;
+import edu.wpi.first.wpilibj.shuffleboard.BuiltInWidgets;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import frc.robot.widgets.MotorWidget;
+import frc.robot.widgets.SCGWidget;
 import io.github.oblarg.oblog.annotations.Config;
 import io.github.oblarg.oblog.annotations.Log;
 
 public class Chassis {
-    @Log.SpeedController
-    private WPI_TalonSRX motorFL;
-    @Log.SpeedController
-    private WPI_TalonSRX motorFR;
+    // @Log.SpeedController
+    // private WPI_TalonSRX motorFL;
+    // @Log.SpeedController
+    // private WPI_TalonSRX motorFR;
     private CANSparkMax sparkMax1; // TBD
+    private CANPIDController sparkMax1PID;
     private CANSparkMax sparkMax2;
+    private CANPIDController sparkMax2PID;
+    private CANSparkMax sparkMax3;
+    private CANPIDController sparkMax3PID;
+    private CANSparkMax sparkMax4;
+    private CANPIDController sparkMax4PID;
+    
     private DifferentialDrive driveTrain;
 
+    private ShuffleboardTab tab;
+
+    private MotorWidget widget1;
+    private MotorWidget widget2;
+    private MotorWidget widget3;
+    private MotorWidget widget4;
+    private SCGWidget widget5;
+    private SCGWidget widget6;
```

Comment:
You might consider opening an issue to help us remember to convert this to Oblog at a later time.